test/e2e/scrapeconfig_test.go

```diff
@@ -90,7 +90,7 @@ func testScrapeConfigCreation(t *testing.T) {
 			spec: monitoringv1alpha1.ScrapeConfigSpec{
 				KubernetesSDConfigs: []monitoringv1alpha1.KubernetesSDConfig{
 					{
-						Role: "Node",
+						Role: "node",
```

Comment:
This shouldn't be needed since we convert to Lowercase always https://github.com/prometheus-operator/prometheus-operator/blob/3c8a018721044c3ced2eaa4c56f9ff9be3d94f31/pkg/prometheus/promcfg.go#L2375-L2382